doc/sphinx/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
why did you use a MakeFile? (Did you write it yourself/ did you follow a tutorial?

Reply:
Yes, the project structure and the makefiles all generated by sphinx-quickstart.

Reply:
maybe we can add a comment on the top :)